piptools/repositories/pypi.py

```diff
@@ -62,6 +63,8 @@ def __init__(self, pip_args, cache_dir, build_isolation=False):
         # and pre) are deferred to pip.
         self.command = create_install_command()
         self.options, _ = self.command.parse_args(pip_args)
+        if self.options.cache_dir:
```

Comment:
This conditional is always `True`, that's why patch test coverage is not 100%. Can we add one more test?

```python
def test_pip_cache_dir_is_empty(from_line, tmpdir):
    pypi_repository = PyPIRepository(
        ["--no-cache-dir"], cache_dir=str(tmpdir / "pypi-repo-cache")
    )

    assert pypi_repository.options.cache_dir is False
```

Reply:
hmmm, does not seem to be much of help.
@atugushev do you understand why does it consider coverage as decreasing?

Reply:
We usually ignore `codecov/project` check. Most important is:

```
codecov/patch — 100.00% of diff hit (target 99.26%)
```

Which is good now :)